keel-core/src/main/kotlin/com/netflix/spinnaker/keel/serialization/ObjectMappers.kt

```diff
@@ -27,7 +28,9 @@ fun configuredObjectMapper(): ObjectMapper =
  * Factory method for [YAMLMapper]s configured how we like 'em.
  */
 fun configuredYamlMapper(): YAMLMapper =
-  YAMLMapper().configureMe()
+  YAMLMapper()
+    .configureMe()
+    .disable(USE_NATIVE_TYPE_ID)
```

Comment:
This is somewhat unrelated. It fixes this:
```yaml
  deployWith: !<red-black>
```
...into this:
```yaml
  deployWith:
    strategy: "red-black"
```

Reply:
Where was that a problem? The manifest we use for keel itself already uses the latter format.

Reply:
In the export API output.